src/js/view/view.js

```diff
@@ -527,8 +527,11 @@ function View(_api, _model) {
             _toggleDOMFullscreen(_playerElement, state);
         } else {
             // Request media element fullscreen (iOS)
-            const provider = model.getVideo();
-            provider.setFullscreen(state);
+            const instreamProvider = model.get('instream') ? model.get('instream').provider : null;
```

Comment:
I think `model.getVideo` already contains this logic.

Reply:
It just returns the `_provider`
https://github.com/jwplayer/jwplayer/blob/master/src/js/controller/model.js#L76

Reply:
Can you just do one `get`?